lexer/lexer.go

```diff
@@ -72,12 +72,18 @@ func (l *Lexer) NextToken() *token.Token {
 		} else {
 			return token.EOFT
 		}
+	case '.':
+		if nextChar == '.' { // DOTDOT
+			l.pos++
+			return token.ConstantTokenChar2(ch, nextChar)
+		}
+		// number can start with . eg .5
+		return token.Intern(l.readNumber(ch))
```

Comment:
No check on isDigit before ?

See https://github.com/grol-io/grol/pull/99#discussion_r1703056718

Reply:
could have been "passthrough" but I already know it started with . so I don't want to redo the letter check

Reply:
short version is unlike some of the other scripting language that force you to type `0.5`, grol is fine with `.5` (had to handle that in readNumber, including that it doesn't allow 2 dots etc... see that code and its tests)